### PR TITLE
initialize offboard structs

### PR DESCRIPTION
**Pull request by AndreasAntener**
Fixes the resetting yaw rate setpoint bug (initialization pos_sp_triplet does, the other two are preemptive).

The problem was the disable_mc_yaw_control flag, checks resulted to true.

### CODE REVIEW

File: src/modules/mavlink/mavlink_receiver.cpp [hunk 1]
```diff
@@ -681,7 +681,7 @@ MavlinkReceiver::handle_message_set_position_target_local_ned(mavlink_message_t
 	mavlink_set_position_target_local_ned_t set_position_target_local_ned;
 	mavlink_msg_set_position_target_local_ned_decode(msg, &set_position_target_local_ned);
 
-	struct offboard_control_mode_s offboard_control_mode;
+	struct offboard_control_mode_s offboard_control_mode = {};
 	memset(&offboard_control_mode, 0, sizeof(offboard_control_mode));//XXX breaks compatibility with multiple setpoints
```
Comment (LorenzMeier) [inline src/modules/mavlink/mavlink_receiver.cpp]: You can remove the memset here

File: src/modules/mavlink/mavlink_receiver.cpp [hunk 2]
```diff
@@ -861,7 +861,7 @@ MavlinkReceiver::handle_message_set_actuator_control_target(mavlink_message_t *m
 	struct offboard_control_mode_s offboard_control_mode;
 	memset(&offboard_control_mode, 0, sizeof(offboard_control_mode));//XXX breaks compatibility with multiple setpoints
 
-	struct actuator_controls_s actuator_controls;
+	struct actuator_controls_s actuator_controls = {};
 	memset(&actuator_controls, 0, sizeof(actuator_controls));//XXX breaks compatibility with multiple setpoints
```
Comment (LorenzMeier) [inline src/modules/mavlink/mavlink_receiver.cpp]: You can remove the memset here

**Comment by AndreasAntener** [comment]
I must have been blind

**Comment by LorenzMeier** [comment]
Applied, thanks!

**Pull request closed without merging**